lib/ansible/utils/module_docs_fragments/aci.py

```diff
@@ -83,6 +71,5 @@ class ModuleDocFragment(object):
     type: bool
     default: 'yes'
 notes:
-- Please read :ref:`the ACI guide <aci_guide>` for more detailed information
-  on how to manage your ACI infrastructure using Ansible.
+- Please read :ref:`aci_guide` for more detailed information on how to manage your ACI infrastructure using Ansible.
```

Comment:
Does that work correctly?
Just checking this doesn't get escaped and ends up not being valid RST

Reply:
This works and the link gets replaced with the title.